Base SDK changed to 2.6.0

junexunxp opened the pull request:
Add mebedtls, add smart washing machine demo application
Add command line input

Commit:
Base SDK changed to 2.6.0
Add mebedtls, add smart washing machine demo application
Add command line input

Commit:
Add command line support, AP join and leave function

Commit:
Enable CLI join and factory reset.
Disable the PRINF log in kv read and write operation

Commit:
AP joining using the CLI to input command "c ssid password" after init done, e.g. "c iPhone 12345678"
AP leave using the CLI to input command "f"

Commit:
fix kv set halt problem

xiongyu0523 reviewed (changes requested):
Remove unnecessary space line, commented code, keep minimum changes that meaningful to the fix

xiongyu0523 commented on component/kv/kv_adapt.c:
```diff
@@ -61,7 +61,7 @@ int32_t kv_flash_write(uint32_t offset, void *buf, uint32_t nbytes)
     
     uint32_t old_primask = DisableGlobalIRQ();
-    
+    //taskENTERCRITICAL();
```
remove

xiongyu0523 commented on component/kv/kv_adapt.c:
```diff
@@ -135,12 +137,13 @@ int32_t kv_flash_write(uint32_t offset, void *buf, uint32_t nbytes)
         DCACHE_InvalidateByRange(FlexSPI_AMBA_BASE + phy_address, FLASH_PAGE_SIZE_BYTES);
     }
-    
+    EnableGlobalIRQ(old_primask);
```
only this line is required

xiongyu0523 reviewed (commented):
1) The commit log is nothing to do with your changes, avoid this kind of commit
2) Should not change the project file for customized settings (if you use CMSIS-DAP, you keep this setting locally, I assume J-Link is by default)

Commit:
Change settings of scanf and printf to support float types
Remove unused items in kv_adapt.c
Close the Mbedtls log output
Add timer functions to washing_machine.c to support reservation timer and local timer events setting and management.
Optimize the command line joining procedure, the output hint will be printed in 8 second interval now.
